backend/core/src/listings/types/listing-application-drop-off-address-type.ts

```diff
@@ -0,0 +1,4 @@
+export enum ListingApplicationDropOffAddressType {
+  leasingAgent = "leasingAgent",
+  mailingAddress = "mailingAddress",
+}
```

Comment:
This and the one below can be shared with `ListingApplicationAddressType` or just `AddressType` is probably fine.